src/card-info.css

```diff
@@ -0,0 +1,129 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+}
+
+.card__img--position {
+  margin: 32px 19px 40px;
+}
+
+.card__item {
+  margin: 0 16px 16px;
+}
+
+.card__info {
+  margin-bottom: 16px;
+}
+
+.card__info-name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 150%;
+  color: var(--color-item-text);
+  margin-bottom: 4px;
+}
+
+.card__info-code {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 140%;
+  color: var(--colot-item-details);
+  margin-bottom: 16px;
+}
+
+.card__star-reviews {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__star {
+  display: flex;
+}
+
+.card_star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  border-radius: 0.5px;
+  margin-left: 4px;
+}
+
+.card_star:first-child {
+  margin-left: 0;
+}
+
+.card__star--1 .card_star:nth-child(1),
+.card__star--2 .card_star:nth-child(-n+2),
+.card__star--3 .card_star:nth-child(-n+3),
+.card__star--4 .card_star:nth-child(-n+4),
+.card__star--5 .card_star:nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__reviews {
+  font-size: 10px;
+  font-weight: 400;
+  color: var(--color-item-text);
+}
+
+.card__cost {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__cost-word {
+  font-size: 12px;
+  line-height: 150%;
+  font-weight: 400;
+  font-style: normal;
+  color: var(--colot-item-details);
+}
+
+.card__cost-sum {
+  font-size: 16px;
+  line-height: 112%;
+  font-weight: 700;
+  font-style: bold;
+  color: var(--color-item-text);
+}
+
+.card__button-buy {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-decoration: none;
+  font-family: inherit;
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+  color: #fff;
+  text-transform: uppercase;
+  width: 166px;
+  height: 40px;
+  background: var(--color-button);
+  border-radius: 5px;
+  border: none;
+}
+
+.card__button-buy:hover {
+  color: var(--color-button);
+  background: #fff;
+  border: 1px solid var(--color-button);
```

Comment:
move this border style directly to button and you don't need to change it on hover